### PR TITLE
chore: update EC config for MAINNET

### LINKED ISSUE #10194: Update hardcoded EC addresses with newly elected committee members
The 3 members and their addresses are:

    "Chloe White": "agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5"
        "Chris Berg": "agoric13p9adwk0na5npfq64g22l6xucvqdmu3xqe70wq"
        "Joe Clark": "agoric1zayxg4e9vd0es9c9jlpt36qtth255txjp6a8yc"

See comment below for more details

### PR DESCRIPTION
**Pull request by rabi-siddique**
closes: #10194 

## Description

The PR updates the `MAINNET` config for the replace electorate core eval by adding new EC member's wallet addresses to `voterAddresses` and updating `highPrioritySendersConfig` with `addressesToRemove`.

### CODE REVIEW

File: packages/builders/scripts/inter-protocol/replace-electorate-core.js [hunk 1]
```diff
@@ -38,18 +38,17 @@ export const defaultProposalBuilder = async ({ publishRef, install }, opts) => {
...
   MAINNET: {
     committeeName: 'Economic Committee',
-    // UNTIL https://github.com/Agoric/agoric-sdk/issues/10194
     voterAddresses: {
-      gov1: 'agoric1gx9uu7y6c90rqruhesae2t7c2vlw4uyyxlqxrx',
-      gov2: 'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
-      gov3: 'agoric14543m33dr28x7qhwc558hzlj9szwhzwzpcmw6a',
+      gov1: 'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
+      gov2: 'agoric13p9adwk0na5npfq64g22l6xucvqdmu3xqe70wq',
+      gov3: 'agoric1zayxg4e9vd0es9c9jlpt36qtth255txjp6a8yc',
```
Comment (dckc) [inline packages/builders/scripts/inter-protocol/replace-electorate-core.js]: The keys should be the names, not gov1 / gov2 / gov3.

https://github.com/Agoric/agoric-sdk/blob/e1baf4e7e8efaab527d7eadf7305281dac8656c9/packages/vm-config/decentral-main-vaults-config.json#L165-L171

**Review by dckc — CHANGES_REQUESTED** [review]
keys should be names for MAIN

to close #10194 , let's finalize all the configs, please - a3p and devnet should have the respective gov4 and a total of 3 after rotation.

**Review by dckc — APPROVED** [review]
I'd rather finish devnet and a3p configs in this pr, but I guess we can do it later

**Comment by rabi-siddique** [comment]
> I'd rather finish devnet and a3p configs in this pr, but I guess we can do it later

Do we need to update the devnet config again? I think we made an update to it in https://github.com/Agoric/agoric-sdk/pull/10251

As for the a3p config, Fraz is adding that in #10241

**Pull request merged**